### PR TITLE
Add includeRestricted to RecommendedAnime Dashboard Panel

### PR DESCRIPTION
**Pull request by natyusha**
- requires api update to function

### CODE REVIEW

**Comment by natyusha** [comment]
> uhh.. you forgetting something? 😔

As a matter of fact, I forgot almost everything :joy:

**Comment by harshithmohan** [comment]
image
You didn't update the type..
The build job is supposed to catch that but didn't. It needs to be fixed...

File: src/core/rtkQuery/splitV3Api/seriesApi.ts [hunk 1]
```diff
@@ -149,7 +149,7 @@ const seriesApi = splitV3Api.injectEndpoints({
     }),
 
     // Gets anidb recommendation for the user
-    getAniDBRecommendedAnime: build.query<SeriesRecommendedType[], PaginationType>({
+    getAniDBRecommendedAnime: build.query<SeriesRecommendedType[], RecommendedPaginationType>({
```
Comment (harshithmohan) [inline src/core/rtkQuery/splitV3Api/seriesApi.ts:152]:
```suggestion
    getAniDBRecommendedAnime: build.query<SeriesRecommendedType[], PaginationType & { includeRestricted: boolean; }>({
```

I don't see a point on adding a whole another type if it's not going to be used elsewhere. This should be just fine. Remove the type declaration

**Review by harshithmohan — CHANGES_REQUESTED** [review]

File: src/core/rtkQuery/splitV3Api/seriesApi.ts (hunk 1 above)
Comment (natyusha) [reply src/core/rtkQuery/splitV3Api/seriesApi.ts]: Applied the requested changes

**Review by harshithmohan — APPROVED** [review]

**Pull request merged**